integration_tests/test_binary_compatibility.py

```
@@ -0,0 +1,561 @@
+                    ]
+                    
+                    for pattern, description in patterns:
+                        match = re.search(pattern, log_content, re.IGNORECASE)
+                        if match:
+                            has_error = True
+                            # Extract the full error line
+                            error_line = match.group(0)
+                            # Try to get more context
+                            start = max(0, match.start() - 200)
+                            end = min(len(log_content), match.end() + 200)
+                            context = log_content[start:end]
```

Flake8 (7.3.0)

[error] local variable 'context' is assigned to but never used

(F841)